env_pkmn.py

```diff
@@ -42,7 +31,8 @@ def scrape_input(self):
                 break
             temp = self.proc.readline().decode()
             retval += temp
+        if "error" in retval:
+            print(retval)
+            print(crash)
```

Comment:
There are better ways to crash than printing an undefined variable. Try:

    throw new Exception("...some helpful statement here...")

Reply:
Not quite verbatim, but done